packages/taquito-utils/src/constants.ts

```diff
@@ -45,6 +46,7 @@ export const prefix = {
   [Prefix.TZ3]: new Uint8Array([6, 161, 164]),
   [Prefix.KT]: new Uint8Array([2, 90, 121]),
   [Prefix.KT1]: new Uint8Array([2, 90, 121]),
+  [Prefix.TXR1]: new Uint8Array([2, 90, 121]),
```

Comment:
I am wondering if the prefix should rather be [1, 128, 120, 031]. See [TXR1](https://gitlab.com/tezos/tezos/-/blob/master/src/proto_alpha/lib_protocol/tx_rollup_prefixes.ml#L35)

Reply:
you are right, I fixed it and added other definition from rollups

Thank you  =)